components/jaggery-core/org.jaggeryjs.jaggery.core/src/main/java/org/jaggeryjs/jaggery/core/pojos/HostObject.java

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Remove these classes from package pojo, and place it in the root level of the package. Ie, org.jaggeryjs.jaggery.core.